src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -92,12 +97,33 @@ public bool HasContainer
 					SetupContainer();
 				else
 					RemoveContainer();
+
+				UpdateValue(nameof(IViewHandler.ContainerView));
```

Comment:
The HasContainer property adds/removes the container and then calls the ContainerView property mapper.